src/components/messages/CarouselMessage.vue

```diff
@@ -200,16 +191,22 @@ export default {
       return sanitizeHtml(text);
     },
     additionalCardProcessing() {
-      let maxCardHeight = 0;
-      for (let card of this.$refs.cards) {
-        maxCardHeight = card.clientHeight > maxCardHeight ? card.clientHeight : maxCardHeight;
-      }
+  let maxCardHeight = 0;
```

Comment:
While this does remove the warning - it is not I think the problem - the problem is that the cards ref has been removed from the element.
So the fix would be either:
* Add the ref=cards back (if this functionality is useful)
* Remove all this functionality completely (if it is not)

Reply:
removed